src/composables/transactionTx.ts

```diff
@@ -89,6 +89,11 @@ export function useTransactionTx({
 
     return !!(
       innerTx.value?.contractId
+      && innerTx.value?.function
+      && (
+        Object.values(FUNCTION_TYPE_DEX).flat()
```

Comment:
I suggest to use `Object.values(TX_FUNCTIONS)` here.

Reply:
We are searching only for DEXfunctions, but `TX_FUNCTIONS` contain all function not only DEX.

Reply:
Uhhh, OK. Thanks for clarification.